lib/flex_commerce_api/json_api_client_extension/requestor.rb

```diff
@@ -3,21 +3,17 @@ module JsonApiClientExtension
     class Requestor < ::JsonApiClient::Query::Requestor
       # expects a record
       def create(record)
-        request(
-          :post,
-          klass.path(record.attributes, record),
-          body: { data: record.as_json_api },
-          params: record.request_params.to_params
-        )
+        request(:post, klass.path(record.attributes), {
+            body: { data: record.as_json_api },
+            params: record.request_params&.to_params
```

Comment:
Actually is request_params coming as nil is acceptable here?

Reply:
Yes because it doesn't look like we are using it but I don't want to remove it for all cases.